doc/manual.xml.head

```diff
@@ -7612,6 +7620,11 @@ set index_format="%4C %Z %{%b %d} %-15.15L (%?l?%4l&amp;%4c?)%*  %s"
           pattern with <quote>^</quote>. This example matches all mails which
           only has recipients from Germany.
         </para>
+        <para>
+          *****)
```

Comment:
Is `*****)` intended?

Reply:
I thought the footnote was useless, so I folded it back to where it was referenced from.

Really ready now.